[IMP] helpdesk: updated teams and stages setup content (#3503)
odoo/documentation: content/applications/services/helpdesk/overview/getting_started.rst

=============================
Getting started with Helpdesk
=============================

> Odoo :guilabel:`Helpdesk` is a ticket based customer support application. Teams can track,

Reviewer: Adjectives like this receive a hyphen.
```suggestion
Odoo :guilabel:`Helpdesk` is a ticket-based customer support application. Teams can track,
```

> prioritize, and solve customer issues from their pipeline, which is organized in customizable
> stages. Multiple teams can be configured and managed in one dashboard.
> 

Reviewer: Remove extra line break.
```suggestion
prioritize, and solve customer issues from their pipeline, which is organized in customizable
stages. Multiple teams can be configured and managed in one dashboard.
```

Setting up teams
================

Setting up multiple teams allows for tickets to be grouped by location or by support type.

> To view or modify Helpdesk teams, go to :menuselection:`Helpdesk --> Configuration --> Teams`.
> To create a new team, click on the :guilabel:`New` button in the top left of the dashboard. From
> there, name the new team, and fill out the remaining fields as defined in the following sections.

Reviewer: 100th char line breaks + finish the last sentence.
```suggestion
To view or modify *Helpdesk *teams, go to :menuselection:`Helpdesk --> Configuration --> Teams`. To
create a new team, click on the :guilabel:`New` button in the top left of the dashboard. From there,
name the new team, and fill out the remaining fields as defined in the following sections on the
form.
```

.. image:: getting_started/helpdesk-teams-list.png
   :align: center
   :alt: View of the helpdesk teams page in Odoo Helpdesk

Assignment & Visibility
-----------------------

Determine to whom the team will be visible
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Under the :guilabel:`Visibility` section, determine who can view this team and its tickets.

- :guilabel:`Invited internal users` have access to the team and tickets they are following. This
  can be modified on each individual ticket.
- :guilabel:`All internal users` have access to the team and all of its tickets without being a
  follower.
- :guilabel:`Invited portal users and all internal users` have access to the team without being a
  follower. Portal users will only be able to access tickets that they are following.

.. example::
   A `Customer Support` team intended to handle general issues with shipping and product issues
   would have the visibility setting :guilabel:`Invited portal users and all internal users`.
   However, a `Financial Services` team handling tickets related to accounting or tax information
   would only need to be visible to :guilabel:`Invited internal users`.

> 

Reviewer: remove extra line break
```suggestion
```

Automatically assign new tickets
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

When tickets are received, they will need to be assigned to a member of the support team. This can

> be done manually on each ticket individually, or through :guilabel:`Automatic Assignment`.
> Check the box next to :guilabel:`Automatic Assignment` to enable the feature for this team.

Reviewer: 100th char
```suggestion
be done manually on each ticket individually, or through :guilabel:`Automatic Assignment`. Check the
box next to :guilabel:`Automatic Assignment` to enable the feature for this team.
```

.. image:: getting_started/assignment-visibility.png
   :align: center
   :alt: View of a helpdesk team settings page emphasizing the assignment and visibility features
         in Odoo Helpdesk

> 

Reviewer: ```suggestion
```

Select one of the following assignment methods, based on how workload should be allocated across

> team:

Reviewer: ```suggestion
the team:
```

- :guilabel:`Each user is assigned an equal number of tickets` assigns tickets to team members

>    based on total ticket count, regardless on the number of open or closed tickets they are

Reviewer: ```suggestion
   based on total ticket count, regardless of the number of open or closed tickets they are
```

   currently assigned.
- :guilabel:`Each user has an equal number of open tickets` assigned tickets to team members based
   on how many open tickets they are currently assigned. This option is useful for automatically
   delegating a heavier workload to high-performers who tend to close tickets quickly.

> 

Reviewer: ```suggestion
```

Finally, add the :guilabel:`Team Members` who will be assigned tickets for this team. Leave the

> field empty to include all employees who have the proper assignments and access rights configured
> in their user account settings.

Reviewer: ```suggestion
field empty to include all employees who have the proper assignments and access rights configured in
their user account settings.
```

.. note::
   If an employee has time off scheduled in the :guilabel:`Time Off` application, they will not be
   assigned tickets during that time. If no employees are available, the system will look ahead
   until there is a match.

.. seealso::
   - :ref:`Manage users <users/add-individual>`
   - :doc:`Access rights </applications/general/users/access_rights>`

> Setting up kanban stages
> ========================

Reviewer: ```suggestion
Create or modify kanban stages
==============================
```

.. important::
   :doc:`Developer mode</applications/general/developer_mode>` must be activated in order to access
   the stages menu. To activate developer mode go to
   :menuselection:`Settings --> General Settings --> Developer Tools` and click on
   :guilabel:`Activate the developer mode`.

The list view shows an overview of all the stages currently available in Helpdesk. They are listed
in the order they appear in the pipeline. To change the order of the stages, use the arrow buttons
on the left side of the list.

.. tip::
   Change the stage order on the kanban view by dragging and dropping individual columns.

.. image:: getting_started/stages-create-new.png
   :align: center
   :alt: View of the stage list page emphasizing the option to create a new stage

To create a new stage, click on the :guilabel:`New` button in the top left of the dashboard.
Next, choose a name for the new stage, and add a description (though it is not required).
Fill out the remaining fields following the steps below.

.. image:: getting_started/stage-settings.png
   :align: center
   :alt: View of a stage's settings page in Odoo Helpdesk

> 

Reviewer: ```suggestion
```

Add email and SMS templates to stages
-------------------------------------

When an :guilabel:`Email Template` is added to a stage, an email is automatically sent to the
the customer when a ticket reaches the stage. An :guilabel:`SMS Template` will result in an
SMS text message sent to the customer.

.. important::
   SMS Text Messaging is an In-App Purchase (IAP) service that requires prepaid credits to work.
   Refer to :doc:`SMS Pricing and FAQ
   </applications/marketing/sms_marketing/pricing/pricing_and_faq>`
   for additional information.

> 

Reviewer: ```suggestion
```

To select an existing email template, select it from the :guilabel:`Email Template` field. Click on
the arrow key to the right of the field to edit the template.

To create a new template, click the field and begin typing a new template title. Then select
:guilabel:`Create and edit`, and complete the form details.

Follow the same steps to select, edit, or create an :guilabel:`SMS Template`.

.. image:: getting_started/sms-template.png
   :align: center
   :alt: View of an SMS template setup page in Odoo Helpdesk

.. seealso::

>    - :doc:`/applications/general/email_communication/email_template`

Reviewer: ```suggestion
   :doc:`/applications/general/email_communication/email_template`
```

> 

Reviewer: ```suggestion
```

Assign stages to a team
-----------------------

Make a selection in the :guilabel:`Teams` field on the :guilabel:`Stages` form. More than one team

> may be selected, as same stage(s) can be assigned to multiple teams.

Reviewer: ```suggestion
may be selected, since the same stage(s) can be assigned to multiple teams.
```

> 

Reviewer: ```suggestion
```

Getting Started
===============

Helpdesk teams provide customers with support to queries or issues they might encounter while using
a product/service. Therefore, having a system in place where multiple teams can co-exist with their
own customized pipeline, visibility settings, and ticket traceability is essential.

Set up teams
============

Setting up multiple teams allows for tickets to be grouped by location
(example: :abbr:`BE/US(Belgium/United States)` ), or by support type (example: IT, accounting, etc.).

To modify or create teams, go to :menuselection:`Helpdesk --> Configuration --> Teams`.

.. image:: getting_started/helpdesk_teams_list.png
   :align: center
   :alt: View of the helpdesk teams page in Odoo Helpdesk

Assignments and visibility
--------------------------

> Visibility
> ~~~~~~~~~~
> The *Visibility* feature specifies who can view this team:

Reviewer: Heading can be more descriptive.
Also need a line break after the heading in the RST.

For the intro paragraph, since we're directly referencing the UI here then it gets a `:guilabel:`. As well, it's not really a feature as much as it's a _setting_. Lastly, no colon before list items that also have colons; just end the sentence on a period. 

Here's how I would rewrite this whole section.

```suggestion
Visibility
~~~~~~~~~~

Under the :guilabel:`Visibility` subheading, decide who can view this team and their activities.
```

> - :guilabel:`Invited internal users`: Internal users will have access to the team and tickets
>   they are following. This access can be modified on each individual ticket.
> - :guilabel:`All internal users`: All internal users will be able to access the team and all of its
>   tickets without being a follower.
> - :guilabel:`Invited portal users and all internal users`: All internal users can access the team
>   and all of its tickets without being a follower. Portal users will only be able to access tickets
>   that they are following.

Reviewer: General: don't add nested colons after a colon, nor capitalize words after a colon.

Specifically: colons are not needed here and we can cut down on wordiness/redundancy.

```suggestion
- :guilabel:`Invited internal users` have access to the team and tickets they are following. This
  access can be modified on each individual ticket.
- :guilabel:`All internal users` will be able to access the team and all of its tickets without
  being a follower.
- :guilabel:`Invited portal users and all internal users` can access the team and all of its tickets
  without being a follower. Portal users will only be able to access tickets that they are
  following.
```

This guarantees that tickets with sensitive or proprietary information will only be seen by the

> appropriate users.

Reviewer: What is `This`?

What is the most restrictive option out of the three listed here?
Can you give an `.. example::` ?

Getting Started
===============

> Odoo :guilabel:`Helpdesk` is a ticket based customer support platform. Teams can track, prioritize,

Reviewer: Not listed in the review but another thing—
The h1 is in title case and a bit generic, would do this instead:

```
=============================
Getting started with Helpdesk
=============================
```

   :guilabel:`Activate the developer mode`.

The list view shows an overview of all the stages currently available in Helpdesk. They are listed
in the order they appear in the pipeline. To change the order of the stages, use the arrow buttons
on the left side of the list.

.. tip::
   Change the stage order on the kanban view by dragging and dropping individual columns.

.. image:: getting_started/stages-create-new.png
   :align: center
   :alt: View of the stage list page emphasizing the option to create a new stage

To create a new stage, click on the :guilabel:`New` button in the top left of the dashboard.
Next, choose a name for the new stage, and add a description (though it is not required).
Fill out the remaining fields following the steps below.

.. image:: getting_started/stage-settings.png
   :align: center
   :alt: View of a stage's settings page in Odoo Helpdesk

Add email and SMS templates to stages
-------------------------------------

When an :guilabel:`Email Template` is added to a stage, an email is automatically sent to the
the customer when a ticket reaches the stage. An :guilabel:`SMS Template` will result in an
SMS text message sent to the customer.

> .. important::
>    SMS Text Messaging is an In-App Purchase (IAP) service that requires prepaid credits to work.
>    Refer to :doc:`SMS Pricing and FAQ
>    </applications/marketing/sms_marketing/pricing/pricing_and_faq>`
>    for additional information.

Reviewer: ```suggestion
.. important::
   SMS Text Messaging is an In-App Purchase (IAP) service that requires prepaid credits to work.
   Refer to :doc:`SMS Pricing and FAQ
   <applications/marketing/sms_marketing/pricing/pricing_and_faq>` for additional information.
```

To select an existing email template, select it from the :guilabel:`Email Template` field. Click on
the arrow key to the right of the field to edit the template.

> To create a new template, click the field and begin typing a new template title. Then select
> :guilabel:`Create and edit`.

Reviewer: ```suggestion
To create a new template, click the field and begin typing a new template title. Then select
:guilabel:`Create and edit`, and fill out the form details.
```

Follow the same steps to select, edit, or create an :guilabel:`SMS Template`.

.. image:: getting_started/sms-template.png
   :align: center
   :alt: View of an SMS template setup page in Odoo Helpdesk

> Assigning stages to a team
> --------------------------

Reviewer: ```suggestion
Assign stages to a team
-----------------------
```

> Make a selection in the :guilabel:`Teams` field. More than one team may be selected, as stages
> can be shared among multiple teams.

Reviewer: ```suggestion
Make a selection in the :guilabel:`Teams` field of the :guilabel:`Stages` form. More than one team
may be selected, as the same stage(s) can be assigned across multiple teams.
```

 .. image:: getting_started/stages-settings-sharing.png
   :align: center
   :alt: View of stage setup emphasizing teams field

> .. example::
>    In the setup below, the stages `New`, `In Progress`, and `Canceled` are shared between the
>    `Customer Care` and `VIP Support` teams. The other teams are only available to one of the two
>    teams.
> 
>   .. image:: getting_started/stages-sharing.png
>    :align: center
>    :alt: View of stages list in Helpdesk showing shared stages

Reviewer: Redundant / don't need.
Also, images are supplemental to the text, and the text should stand on it's own to provide instruction; text should not rely on images to convey important information.
```suggestion
```

Fold a stage
------------

> Check the :guilabel:`Folded in Kanban` box to display this stage as *folded* by default in the
> kanban view for this team.
> 
> .. warning::
>    Tickets that reach a *folded* stage are considered closed. Closing a ticket before the work is
>    completed can result in reporting and communication issues. This setting should only be enabled
>    for stages that are considered *closing* stages.
> 
> .. image:: getting_started/folded-closing.png
>    :align: center
>    :alt: View of the 'Folded in Kanban' setting enabled
> 
> .. tip::
>    Stages can be temporarily folded in the kanban view, as seen below. This will not close the
>    tickets in this stage.
> 
> .. image:: getting_started/folded-in-pipeline.png
>    :align: center
>    :alt: View of Fold setting in Kanban pipeline

Reviewer: Reduce unnecessary scrolling for the reader by:
- condensing similar information together
- being specific about location and instructions
- removing unnecessary images

```suggestion
Check the :guilabel:`Folded in Kanban` box on the :guilabel:`Stages` form to display this stage as
*folded* by default in the kanban view for this team.

.. warning::
   Tickets that reach a *folded* stage are considered closed. Closing a ticket before the work is
   completed can result in reporting and communication issues. This setting should only be enabled
   for stages that are considered *closing* stages.

Alternatively, stages can be temporarily folded in the kanban view, by clicking on the settings icon
and selecting :guilabel:`Fold`.

.. note::
   Manually folding the stage from the kanban view will not close the tickets in this stage.
```
